cordova-lib/spec-cordova/plugin.spec.js

```diff
@@ -203,6 +204,23 @@ describe('plugin end-to-end', function() {
         .fin(done);
     });
 
+    it('should respect preference default values', function (done) {    
+       addPlugin(path.join(pluginsDir, org_test_defaultvariables), org_test_defaultvariables, {cli_variables: { REQUIRED:'NO', REQUIRED_ANDROID:'NO'}}, done)
+       .then(function() {
+            var platformJsonPath = path.join(project, 'plugins', helpers.testPlatform + '.json');
+            var installed_plugins = JSON.parse(fs.readFileSync(platformJsonPath)).installed_plugins;

```

Comment:
Instead of `JSON.parse` & `fs.readFileSync`, why not use 
```var installed_plugins = require('platformJsonPath').installed_plugins;```

That would remove the need to `require('fs')`